user/api.go

```diff
@@ -388,6 +388,9 @@ func (a *Api) UpdateUser(res http.ResponseWriter, req *http.Request, vars map[st
 	} else if originalUser == nil {
 		a.sendError(res, http.StatusUnauthorized, STATUS_UNAUTHORIZED, "User not found")
 
+	} else if originalUser.HasRole(RoleBrokered) {
```

Comment:
More of a nitpick and something to keep in the mind for the future. Idiomatic go usually avoids long if else else if statements in favor of either switch statements or early return in their own if clauses
https://go.dev/doc/effective_go#switch
https://go.dev/doc/effective_go#if

Reply:
I know, unfortunately it's kinda difficult to use idiomatic go in this codebase without changing too much code, which doesn't have almost any meaningful tests. Given that this one will be retired soon, it's not worth investing time.